backend/ohq/management/commands/populate.py

```diff
@@ -0,0 +1,458 @@
+                        "video_chat_url": "https://upenn.zoom.us/j/armaanTakeMyChildren",
+                        "status": Question.STATUS_ACTIVE,
+                        "time_asked": now - datetime.timedelta(minutes=400),
+                        "time_response_started": now - datetime.timedelta(minutes=130),
+                        "should_send_up_soon_notification": False,
+                    },
+                ],
+            },
+            {
+                "name": "Other Questions",
+                "description": "Questions for not so good vibes",
+                "archived": False,
+                "estimated_wait_time": 30,
+                "active": True,
+                "questions": [
+                    {
+                        "text": "Potatoes",
+                        "video_chat_url": "https://upenn.zoom.us/j/ad123456sdfghd13",
+                        "status": Question.STATUS_WITHDRAWN,
+                        "time_asked": now - datetime.timedelta(minutes=1),
+                        "should_send_up_soon_notification": False,
+                    },
+                    {
+                        "text": "How's life at the Labs?",
+                        "video_chat_url": "https://upenn.zoom.us/j/daaaarrkkkkkMooodddeee",
+                        "status": Question.STATUS_ANSWERED,
+                        "time_asked": now - datetime.timedelta(minutes=240),
+                        "time_response_started": now - datetime.timedelta(minutes=50),
+                        "time_responded_to": now - datetime.timedelta(minutes=17),
+                        "should_send_up_soon_notification": True,
+                    },
+                ],
+            },
+            {
+                "name": "We are archiving the bad vibes",
+                "description": "No bad vibes allowed",
+                "archived": True,
+                "estimated_wait_time": 0,
+                "active": False,
+                "questions": [],
+            },
+        ],
+        "semester": {"year": 2020, "term": Semester.TERM_SPRING},
+        "archived": False,
+        "invite_only": True,
+        "video_chat_enabled": False,
+        "require_video_chat_url_on_questions": True,
+    },
+    {
+        "course_code": "500",
+        "department": "OLD",
+        "course_title": "Class of 1930",
+        "description": "Class of 1930 Graduates Only! Likely to be archived",
+        "semester": {"year": 1930, "term": Semester.TERM_SPRING},
+        "archived": True,
+        "queues": [],
+        "invite_only": False,
+        "video_chat_enabled": True,
+        "require_video_chat_url_on_questions": True,
+    },
+]
+
+
+class Command(BaseCommand):
+    help = "Populates the development environment with dummy data."
+
+    def handle(self, *args, **kwargs):
+
+        # not sure if this line is neccessary / works
+        if Course.objects.filter(course_title="Example Queues for OHQ").exists():
```

Comment:
You should be able to use some env var (I think settings.DEBUG?) to check whether you are in production isntead of relying on a specific course